R/generate_client.R

```diff
@@ -98,17 +98,27 @@ build_function_args <- function(params) {
   paste0(" <- function(", arg_str, ") {\n\n")
 }
 
+#' @importFrom utils tail
 build_function_name <- function(verb_name, path_name) {
   parts <- strsplit(path_name, "/")[[1]]
-  parts <- parts[parts != ""]
-  parts <- parts[!startsWith(parts, "{")]
+  parts <- parts[parts != '']
   parts <- gsub("-", "_", parts)
-
+  args <- tail(parts, -1)
```

Comment:
This is a transliteration from the python client here: https://github.com/civisanalytics/civis-python/blob/c220a5244573e37f5687138059029f8c0bc7b957/civis/resources/_resources.py#L400

Reply:
This is related to https://github.com/civisanalytics/civis-python/issues/298 correct? Does this fix that issue?

Reply:
Yep! It might not be an exact transliteration.